common/ManagementTestShared/Redesign/JsonHelper.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The folder seems odd here - why the 'Redesign' folder name?

Reply:
it separates the current preview track2 includes from the new track2 includes.  At some point this can disappear, but we just needed two folders.